database/README.md

```diff
@@ -353,6 +357,14 @@ WHERE
 LIMIT 10000;
 ```
 
+### Creating a new geospatial layer
+
+When creating a new database table to hold feature data, polygon geometries should always be stored in a `Multipolygon` column type to avoid future issues. Any columns that will be populated with AGOL feature attribute data should exactly match the column names used in AGOL, except they should be lowercase.
+
+As a best practice, tables should always be configured with `created_at` and `updated_at` columns of type `timestamptz` with default `now()`. As well, the layer should be configured with an `on update` trigger to set the `updated_at` timestamp using the existing `set_updated_at_timestamp()` function.
+
+Lastly, new layers should make use of the [ArcGIS Online Layer Helper](/toolbox/load_agol_layer) tool so that they can be easily refreshed/maintained.
+
```

Comment:
i have failed to discuss in these docs what we should do to refresh values in the crashes table when one of the reference layers is updated. E.g., if we re-load the council districts into the database, we need to re-trigger the `crashes_set_spatial_attributes` function to update all crashes accordingly.

I'm going to address this when we do to the upcoming work to configure triggers for signal engineer areas and non-coa roadways. (i'll link back with an issue shortly).